model_bakery/baker.py

```diff
@@ -687,8 +691,20 @@ def _handle_m2m(self, instance: Model):
                     make(through_model, _using=self._using, **base_kwargs)
 
     def _handle_generic_foreign_keys(self, instance: Model, attrs: Dict[str, Any]):
-        for key, value in attrs.items():
-            setattr(instance, key, value)
+        """Set content type and object id for GenericForeignKey fields."""
+        for _field_name, data in attrs.items():
+            content_type_field = data["content_type_field"]
+            object_id_field = data["object_id_field"]
+            value = data["value"]
+            if is_iterator(value):
+                value = next(value)
+
+            setattr(
+                instance,
+                content_type_field,
+                contenttypes_models.ContentType.objects.get_for_model(value),
+            )
+            setattr(instance, object_id_field, value.pk)
```

Comment:
After this change, null values for a generic foreign key (where both the `content_type` and `object_id` fields are nullable) fail here. I believe this section should be skipped when `value` is `None`.

Reply:
Hm, you are right!

I have drafted #496 to address that, however it would not prevent `content_type` and `object_id` from generating random values in case they are not nullable and `content_object` is None.
(This was never a case before either, so I am not sure how critical it is to cover now - as it is not easy).

Reply:
Released 1.19.3 adjusting this:
- https://github.com/model-bakers/model_bakery/releases/tag/1.19.3
- https://pypi.org/project/model-bakery/1.19.3/